packages/nocodb/src/db/sql-client/lib/pg/YugabyteClient.ts

```diff
@@ -114,7 +119,7 @@ class YBClient extends PGClient {
         // column['unique'] = response.rows[i]['cst'].indexOf('UNIQUE') === -1 ? false : true;
 
         column.cdf = response.rows[i].cdf
-          ? response.rows[i].cdf.split('::')[0].replace(/'/g, '')
+          ? response.rows[i].cdf.replace(/::[\w ]+$/, '').replace(/^'|'$/g, '')
```

Comment:
type can contain scale, precision etc. May be `.replace(/::[\w ]+$/, '')` won't be enough.

for example:

```
value::NUMERIC(1,2)
```

Reply:
I replaced regex with /::[\w (),]+$/ to support scale/precision as well (both pg and yugabyte)
Let me know if this is enough (I am trying to avoid split bc that way we need to handle :: in value as well, with regex we are making sure it is on ending)

Reply:
Looks fine now